plugins/modules/ipahostgroup.py

```diff
@@ -315,6 +391,19 @@ def main():
                                          "host": host,
                                          "hostgroup": hostgroup,
                                      }])
+
+                    if has_add_membermanager:
+                        # Remove membermanager users and groups
+                        if len(membermanager_user) > 0 or \
+                           len(membermanager_group) > 0:
```

Comment:
These lines show the same issue as before, just add state: absent to the test playbook.

I'd suggest the same fix here.